crates/biome_lsp_converters/src/lib.rs

```diff
@@ -127,14 +72,6 @@ mod tests {
         check_conversion!(line_index: Position { line: 1, character: 3 } => TextSize::from(7));
     }
 
-    #[test]
-    fn out_of_bounds_line() {
-        let line_index = LineIndex::new("abcde\nfghij\n");
-
-        let offset = line_index.offset(LineCol { line: 5, col: 0 });
-        assert!(offset.is_none());
-    }
```

Comment:
This lives in `biome_line_index` because it specifically tests `LineIndex`, not an LSP related thing